kafka/consumer/group.py

```diff
@@ -462,7 +462,7 @@ def _poll_once(self, timeout_ms):
             return records
 
         self._fetcher.init_fetches()
-        self._client.poll(timeout_ms)
+        self._client.poll(timeout_ms, sleep=True)
```

Comment:
Thanks for the thoughts. Can you remove from this PR? This is a separate issue (and also I believe the incorrect fix for the bug you described).